Makefile

```diff
@@ -41,12 +46,17 @@ apply-crd:
 	@kubectl delete crd loadtests.kangal.hellofresh.com || true
 	@kubectl apply -f charts/kangal/crd.yaml
 
-protoc:
+# Transpile proto file(s) to source code
+protoc: $(tools/protoc) $(tools/protobuf) $(tools/protoc-gen-go) $(tools/protoc-gen-go-grpc) $(tools/protoc-gen-grpc-gateway) $(tools/protoc-gen-openapiv2) $(tools/gateway)
```

Comment:
```suggestion
protoc: tools
```

Reply:
who knows, maybe we'll need another tools =) but for now will do